### PR TITLE
docs: refresh translation guidance

### PR DESCRIPTION
**Pull request by pesap**
## Summary

Refresh the R2X documentation structure following the documentation enhancement approach used by r2x-cli PR #185.

- Add a concise documentation index and getting-started path.
- Add architecture and development guidance grounded in the current package layout.
- Organize translation workflows and contributor guidance around current public APIs.
- Add explicit instructions for adding, reviewing, and maintaining translated documentation.

## Testing Strategy

- `git diff --check`
- Repository pre-commit hooks passed when the commit was created.

## Stacked PR

This is the first PR in a stack. The Astro migration follows in PR #326.

### CODE REVIEW

File: docs/source/architecture.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    │
+    ▼
+parser package  ──>  r2x_core.System  ──>  R2X translation plugin
+                                                    │
+                                                    ▼
+                                            target r2x_core.System
+                                                    │
+                                                    ▼
+                                             exporter package
+```
```
Comment (pesap) [inline docs/source/architecture.md]: This should be a mermaid diagram instead.

File: docs/source/architecture.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Package boundaries
+
+| Layer | Responsibility | Examples |
+| --- | --- | --- |
+| CLI | Discovers plugins, manages the Python environment, and runs pipelines. | `r2x-cli` |
+| Parser | Reads a source format into an `r2x_core.System`. | `r2x-reeds`, `r2x-plexos`, `r2x-sienna` |
+| Core | Provides the shared `System`, `PluginContext`, `Rule`, and rule-engine APIs. | `r2x-core` |
+| Translation | Maps source components and fields to target components and attaches derived data. | This repository |
+| Exporter | Writes a target system to its native format. | `r2x-plexos`, `r2x-sienna` |
```
Comment (pesap) [inline docs/source/architecture.md]: Add hyperlink to the repos.

File: docs/source/architecture.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Translation package structure
+
+Each package under `packages/` has the same broad shape:
+
+- `translation.py` exposes the public translation function.
+- `plugin_config.py` defines the typed configuration accepted by that function.
+- `config/rules.json` contains declarative source-to-target mappings.
+- `getters.py` and `getters_utils.py` contain derived-field and post-processing logic.
+- `tests/` exercises translation behavior and important edge cases.
```
Comment (pesap) [inline docs/source/architecture.md]: This is missing important artifacts under config. See r2x-core for the latests.

File: docs/source/architecture.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+Rules handle direct field mappings, defaults, source and target component types,
+and filters. Getter functions handle values that require computation or context,
+such as unit conversion, commitment status, outage rates, memberships, and
+name resolution. Keep these responsibilities separate: put a stable mapping in
+`rules.json`, and put computation that needs code or context in a getter.
+
+When a mapping changes, update the rule and the behavior-focused tests together.
+When a getter changes, test both its returned value and the resulting target
+component or time series where feasible.
```
Comment (pesap) [inline docs/source/architecture.md:68]: Hyperlink to r2x-core

File: docs/source/development.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+   file paths, or configuration keys.
+3. Add the translated page to the same sidebar position as the English page and
+   add the language to the site language selector when the site supports one.
+4. Record the English source page and its source revision in the translated
+   page's frontmatter or maintenance metadata. This makes stale translations
+   discoverable after an English update.
+5. Have a reviewer who understands the target language and the R2X workflow
+   verify technical meaning, commands, links, and terminology.
+6. Build the documentation and inspect the rendered translated page before
+   merging.
```
Comment (pesap) [inline docs/source/development.md]: Thrash

File: docs/source/architecture.md (hunk 4 above)
Comment (pesap) [reply docs/source/architecture.md:68]: Addressed in `2ae8dcf`: the page now links to the r2x-core plugin-system and rules-system documentation.

File: docs/source/architecture.md (hunk 1 above)
Comment (pesap) [reply docs/source/architecture.md]: Addressed in `27f55cd`: replaced the text diagram with a Mermaid flowchart.

File: docs/source/development.md (hunk 5 above)
Comment (pesap) [reply docs/source/development.md]: Addressed in `2ae8dcf`: removed the out-of-scope documentation-localization section and reduced the development page to repository checks and interoperability-package contribution guidance.

File: docs/source/getting_started.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+To manage the environment yourself, install a published package with `pip`:
+
+```bash
+python -m pip install r2x-reeds-to-plexos
+```
+
+See the package [README files](https://github.com/NatlabRockies/R2X/tree/main/packages)
+for package-specific entry points and examples.
```
[the remark this answers is not in the document]
Comment (pesap) [reply docs/source/getting_started.md]: Addressed in `27f55cd`: the getting-started page now points readers to the workflow guides rather than the stale package README examples.

File: docs/source/architecture.md (hunk 2 above)
Comment (pesap) [reply docs/source/architecture.md]: Addressed in `2ae8dcf`: the package-boundary table now links to the r2x-cli, parser, r2x-core, and exporter repositories.

File: docs/source/architecture.md (hunk 3 above)
Comment (pesap) [reply docs/source/architecture.md]: Addressed in `2ae8dcf`: the architecture page now documents the package-specific `config/` artifacts, including `defaults.json`, `rules.json`, and configuration helpers.

File: docs/source/development.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+## Add or update a translation
+
+1. Choose the package under `packages/` that owns the source-to-target direction.
+2. Confirm the source and target component APIs in the parser, exporter, and
+   `r2x-core` packages before changing a mapping.
+3. Update `config/rules.json` for direct mappings, defaults, filters, and type
+   changes.
+4. Update `getters.py` or `getters_utils.py` only when the value needs
+   computation, context, unit conversion, membership resolution, or
+   post-processing.
```
[the remark this answers is not in the document]
Comment (pesap) [reply docs/source/development.md]: Addressed in `27f55cd`: the contributor guidance now refers to package-specific getter and post-processing modules instead of assuming a shared filename.

File: docs/source/architecture.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+| Exporter | Writes a target system to its native format. | `r2x-plexos`, `r2x-sienna` |
+
+## Translation package structure
+
+Each package under `packages/` has the same broad shape:
+
+- `translation.py` exposes the public translation function.
+- `plugin_config.py` defines the typed configuration accepted by that function.
+- `config/rules.json` contains declarative source-to-target mappings.
+- `getters.py` and `getters_utils.py` contain derived-field and post-processing logic.
```
[the remark this answers is not in the document]
Comment (pesap) [reply docs/source/architecture.md]: Addressed in `27f55cd`: the architecture page now describes getter and post-processing modules as package-specific and lists the observed naming variants.

File: docs/source/dev_workflow.md [hunk 9]
```diff
@@ -1,7 +1,8 @@
-# Dev Workflow
+# Translation workflows
 
-Each workflow below shows how to install the relevant upstream packages as editable installs
-(so local changes are picked up immediately) and provides a full end-to-end example script.
+Each workflow below shows how to install the upstream parser and exporter as
+editable packages and provides a complete Python API example. Use the guide
+that matches your source and target formats.
```
[the remark this answers is not in the document]
Comment (pesap) [reply docs/source/dev_workflow.md]: Addressed in `27f55cd`: the workflow page now labels these as contributor-oriented integration examples and directs readers seeking the smaller public API surface to Getting started.

File: docs/source/development.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+exactly so readers can search for them and copy examples.
+
+## Documentation build
+
+The Astro site is in `docs/` and uses npm with the committed lockfile. From
+that directory:
+
+```bash
+npm ci
+npm run build
```
[the remark this answers is not in the document]
Comment (pesap) [reply docs/source/development.md]: Addressed in `27f55cd`: restored the current Sphinx and uv docs-group commands used by this PR's CI. The Astro/npm instructions remain scoped to the dependent migration PR.

File: docs/source/architecture.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
+# Architecture
+
+R2X is the translation layer in the broader r2x model-interoperability
+workflow. Each stage has a focused responsibility:
+
+```mermaid
```
Comment (mcllerena) [inline docs/source/architecture.md]: > {mermaid}

Isn't this the right way to identify mermaid?

**Review by mcllerena — CHANGES_REQUESTED** [review]

**Comment by mcllerena** [comment]
LGTM!

**Review by mcllerena — APPROVED** [review]
